Feat: Added Protestant Reformation decision, Localization to be followed

John-Timothy opened the pull request.

John-Timothy committed:
Feat: Added Protestant Reformation decision, Localization to be followed

John-Timothy committed:
Feat: Added Localization for Decision

HansKalsi commented on common/decisions/00_protestant_reformation_decision.txt:
```diff
@@ -0,0 +1,60 @@
+		capital_county = {
+			development_level => 35
+		}
+	}
```
Overall looks great - only thing missing for the is_valid is the requirement that this can only happen within German lands.
This will be a custom region, include any Central Germanic cultures for the custom region - they must be within the region (not control it, be within it)

John-Timothy replied:
Here are the Changes:

`	is_valid = {
		faith.religious_head = {
			opinion = {
				target = root
				value < -100
			}
		}

		capital_county = {
			development_level > 35
		}

		root = {
			any_held_title = {
				title_province = { geographical_region = central_germanic_lands }
			}
		}
	}`

HansKalsi commented on common/decisions/00_protestant_reformation_decision.txt:
```diff
@@ -0,0 +1,60 @@
+				culture = culture:bavarian
+				culture = culture:franconian
+				culture = culture:swabian
+				culture = culture:polabian
```
Polobian is not a germanic culture, just like for the other task, please again refer to this source:
https://ck3.paradoxwikis.com/Culture#:~:text=Original-,Central%20Germanic,-Bavarian

Any areas on the map that start with those cultures in the 1799 start date should be included in the custom region FYI

John-Timothy replied:
`	is_shown = {
		root = {
			OR = {
				culture = culture:saxon
				culture = culture:bavarian
				culture = culture:franconian
				culture = culture:swabian
				culture = culture:dutch
				culture = culture:frisian
				culture = culture:german
			}
		}
	}`

John-Timothy replied:
Here are the changes ^

John-Timothy committed:
Feat: Changed to Orthodox Faith

- Also Checks Every Vassal that has a positive opinion of the player to also change faith
- Commented out the Every Held Title for now but useful for testing effect

John-Timothy committed:
Refactor: Changes to Culture checks and If player has title within region

John-Timothy committed:
Refactor: Conflict Refactoring

- Added German Unification Localization to Protestant Reformation
- Added Custom German Regions to resolve conflicts

John-Timothy committed:
Refactor: Resolving conflicts from Main Branch to Protestant Reformation Branch

HansKalsi committed:
Remove unneeded code leftover from rebase

HansKalsi committed:
- Refined protestant reformation decision
- Added pope character (who never appears for whatever reason)
- Added Protestant religion
- Added Papal State country to 1799 bookmark

HansKalsi committed:
- Added custom decision gfx
- Added logic so protestant reformation decision isn't available if you are protestant
